_includes/install_command.html

```diff
@@ -0,0 +1,49 @@
+{%- capture code -%}
+brew {{include.modifier}} install {{include.item}}
+{%- endcapture -%}
+<style>
+    .inline {
+        display: inline-flex;
+        background: rgba(0, 0, 0, 0.3);
```

Comment:
Would be good to use the existing code here (by adding e.g. a new class to the brew.sh CSS) rather than adding hardcoding the same here.